sdk/cosmosdb/cosmos/src/diagnostics/CosmosDiagnosticsContext.ts

```diff
@@ -53,6 +55,15 @@ export class CosmosDiagnosticContext {
     this.gaterwayStatistics.push(gaterwayStatistics);
   }
 
+  public recordEncryptionDiagnostics(encryptionDiagnostics: EncryptionDiagnostics): void {
+    const { encryptContent, decryptContent } = encryptionDiagnostics;
+    const encryptionDuration = encryptContent[Constants.Encryption.DiagnosticsDuration] ?? 0;
```

Comment:
in what scenarios DiagnosticsDuration will be undefined

Reply:
In operations like change feed for entire container, we would not require to encrypt anything. In that case, encryptContent would be empty.